doc/python_api_reference_vDev.md

```diff
@@ -7659,9 +7713,45 @@ def cnot(
 # (in class stim.TableauSimulator)
 def copy(
     self,
+    *,
+    also_copy_rng_state: bool = False,
+    seed: Optional[int] = None,
 ) -> stim.TableauSimulator:
+
     """Returns a copy of the simulator. A simulator with the same internal state.
 
+    Args:
+        also_copy_rng_state: If False, RNG is reinitialized using a random seed.
+            If True, RNG state is copied together with the simulator. The two
+            copies will then produce the same measurement outcomes for the same
+            quantum circuits. If both also_copy_rng_state and seed are used,
+            an exception is raised. Defaults to False.
+        seed: PARTIALLY determines simulation results by deterministically seeding
+            the random number generator.
+
+            Must be None or an integer in range(2**64).
+
+            Defaults to None. When None, the prng is seeded from system entropy.
```

Comment:
If the user specifies both `alsp_copy_rng_state` and `seed`, there should be an exception. Also, this text here should say that if it's set to None then the `also_copy_rng_state` argument is controlling what happens.

Reply:
Done.